### PR TITLE
Remove throttler hack

### PR DESCRIPTION
**Pull request by mihails-strasuns**

### CODE REVIEW

File: src/ocean/task/Scheduler.d [hunk 1]
```diff
@@ -555,11 +555,18 @@ final class Scheduler : IScheduler
...
+                    "({} worker fibers still suspended, " ~
+                    "{} pending tasks to resume)",
+                this.fiber_pool.num_busy(),
+                this.cycle_pending_task_count
+            );
 
         }
-        while (this.fiber_pool.queued_tasks.length && this.cycle_pending_task_count);
+        while ((this.fiber_pool.queued_tasks.length ||
+            this.cycle_pending_task_count) && this.state != State.Shutdown);
```
Comment (gavin-norman-sociomantic) [inline src/ocean/task/Scheduler.d:569]: Does this change in the scheduler relate to "Make throttling hook run once per epoll cycle"? (Is this a separate change?)

Comment (mihails-strasuns) [reply src/ocean/task/Scheduler.d:569]: Related but separate bug fix. Will split.

**Review by matthias-wende-sociomantic — COMMENTED** [review]
Looks good! I'm happy to approve/merge once you're finished :)

**Review by gavin-norman-sociomantic — APPROVED** [review]

**Comment by gavin-norman-sociomantic** [comment]
(Approved apart from the failed tests, of course.)

**Pull request merged**